packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1096,6 +1096,7 @@ export class ContainerRuntime
 	private imminentClosure: boolean = false;
 
 	private readonly _getClientId: () => string | undefined;
+	private readonly _getConnected: () => boolean;
```

Comment:
Let's have a doc comment here explaining this is getting the true connected state of the Container, irrespective of readonly state (in contrast to the public getter `connected`)

Reply:
Maybe even name it `_getTrueConnectedState` or `_getRawConnectedState`?